Client/ContentBlocker/ContentBlocker+Safelist.swift

```diff
@@ -80,12 +85,27 @@ extension ContentBlocker {
     }
 
     func readSafelistFile() -> [String]? {
-        guard let fileURL = safelistFileURL() else { return nil }
+        if  let fileURL = safelistFileURL(),
+            FileManager.default.fileExists(atPath: fileURL.path) {
+            return readSafelistFile(fileURL: fileURL)
+
+            // read file data, then move to new location
+        } else if let oldFileURL = oldSafelistFileURL(),
+                  FileManager.default.fileExists(atPath: oldFileURL.path) {
+            let data = readSafelistFile(fileURL: oldFileURL)
+            if let newFileURL = safelistFileURL() {
```

Comment:
Instead of having pyramids of `if let`, seeing as you're using this url in both places, then maybe have it as a guard let at the top of the function.